src/Parser/Priest/Shadow/CHANGELOG.js

```diff
@@ -1,7 +1,8 @@
 export default `
-25-11-2017 - Fixed damage increase modifier of Heart of the Void & cooldown on Shadow crash due to class changes.
-15-11-2017 - Added The Twins Painful Touch (Legendary), Anunds Seared Shackles (Legendary), Heart of the Void (Legendary) & Zenkaram Iridis (Legendary).
-04-11-2017 - Fixed ABC downtime.
+04-12-2017 - Added skippable casts. Mastery values added. Twist Of Fate uptime added. Changed layout. Fixed cooldown on Mind blast. (by hassebewlen)
```

Comment:
Please note this changelog format is deprecated. Please upgrade to the new format (see other specs/core) when possible.